sphinx_needs/config.py

```diff
@@ -301,6 +307,16 @@ def get_default(cls, name: str) -> Any:
             return _field.default_factory()
         return _field.default
 
+    from_toml: str | tuple[str, list[str]] | None = field(
+        default=None, metadata={"rebuild": "env", "types": (str, tuple, list)}
+    )
+    """Path to a TOML file to load configuration from.
+    
+    Optionally, a tuple of a path and a path to the section in the toml,
```

Comment:
Was reading the sentence multiple times to understand what the tuple does. One more sentence would be good what tools and needs config leads to.

Reply:
split off the section specification into a separate config option